src/mumble/Messages.cpp

```diff
@@ -1204,6 +1204,14 @@ void MainWindow::msgPermissionQuery(const MumbleProto::PermissionQuery &msg) {
 ///
 /// @param msg The message object
 void MainWindow::msgCodecVersion(const MumbleProto::CodecVersion &msg) {
+	if (!msg.opus()) {
+		qWarning() << "Client was instructed to not use Opus!";
+	}
+
+	// Always use Opus
+	Global::get().bOpus = true;
+
+	return;
```

Comment:
@davidebeatrici do you think this is sufficient for enforcing Opus usage on the client?

Reply:
I think so.